src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/services/CheckpointService.scala

```diff
@@ -42,8 +46,7 @@ class HdfsParameters(
 )
 
 @Service
-class CheckpointServiceImpl @Inject() (userGroupInformationWrapper: UserGroupInformationWrapper)
-    extends CheckpointService {
+class CheckpointServiceImpl @Inject() (hdfsService: HdfsService) extends CheckpointService {
```

Comment:
private lazy val fs = FileSystem.get(conf) not used

Reply:
I knew I forgot something 🤦